src/app/page/UserProfile/UserProfile.js

```diff
@@ -85,6 +85,7 @@ const UserProfile = ({ history }) => {
               addressMapPoint: { latitude: data.lat, longitude: data.long },
             })
           );
+      setSuccessSnackbarOpen(true);
     } catch (error) {
       console.log("ERROR WHEN GETTiNG LOCATION", error);
```

Comment:
Can we add the error snack bar here where we cannot get location and remove this console log?

Reply:
@azmainamin - I can certainly do that, but the issue is there is already <ErrorSnackbar> setup on that page for an existing user account with different error message, and this logic needs to be changed a bit to avoid duplication. Perhaps, it requires a separate issue on it's own ?